### ISSUE TITLE
No Documentations!

### ISSUE THREAD

**Issue by MidoCobra** [sanaani/django-payment-authorizenet#1]
Hi there ,
Excuse me is this project completed?
if so ... i need some help to know how to set it up in my project..
thanks

**Comment by sanaani** [comment]
Yes, this project is in production. Have you checked the readme?
